client/src/components/CreateList/CreateListForm.js

```diff
@@ -25,22 +33,25 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-function CreateListForm() {
+function CreateListForm(props) {
   const [name, setName] = useState('');
 
   const dropZoneFilesRef = useRef(null);
   const dropZoneSetFilesRef = useRef(null);
 
   const classes = useStyles();
 
+  // eslint-disable-next-line no-shadow
+  const { createList, clearErrors, error, loading } = props;
+
+  useEffect(() => {
+    clearErrors(); // Clear previous errors when the component mounts
+  }, [clearErrors]);
+
   const handleSubmit = event => {
     event.preventDefault();
-    // eslint-disable-next-line no-unused-vars
-    const formData = {
-      name,
-      cover: dropZoneFilesRef.current[0],
-    };
-    /* TODO: SUBSCRIBE TO REDUX STORE AND PASS FORMDATA TO ACTION CREATORS */
+
+    createList(name, dropZoneFilesRef.current[0]);
```

Comment:
What is .currrent[0]?

Reply:
`current` is a property of the `useRef` hook. You use it to access the value of the reference. Here I used `useRef` to reference the `DropZone` component's files. The reference is an array as the `DropZone` component can have many files, but we only want the first one.